### PR TITLE
Add unit test for packer build

### PR DESCRIPTION
**Pull request by oxr463**
# Pull request checklist

## Description

- Add packer init and packer build tests.
- Run unit-tests via tox on PR and Push for GitHub Actions.
- Ignore .eggs/ directory.

Reference(s):
- https://github.com/ymyzk/tox-gh-actions#workflow-configuration

## Types of changes

- [ ] This change requires a documentation update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] Lint and unit tests pass locally with my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/test_packer.py [hunk 1]
```diff
@@ -3,28 +3,48 @@
...
+    test_template = './tests/docker.pkr.hcl'
+    packer = Packer(template=test_template)
+
+    @classmethod
+    def setUpClass(self):
+        self.packer.init('tests')
+
     def test_init(self):
+      self.packer.init('tests')
       self.assertEqual("TODO", "FIXME")
```
Comment (oxr463) [inline tests/test_packer.py]: We should check and see if `packer init` creates any files on disk.

Reference(s):
- https://stackoverflow.com/q/59121161/8507637
- https://github.com/hashicorp/packer/blob/main/command/init_test.go

**Comment by oxr463** [comment]
@abhi1693 please review.

**Comment by abhi1693** [comment]
@oxr463 Could you also add a GitHub CI workflow for this to run on every push and pull request?

**Comment by oxr463** [comment]
> @oxr463 Could you also add a GitHub CI workflow for this to run on every push and pull request?

This is done, however, I was expecting tox to fail on all of the unimplemented tests but for some reason it passed.

**Comment by abhi1693** [comment]
@oxr463 Something is not right with the implementation. I don't see any runs or summary of the runs in the tox step

**Comment by abhi1693** [comment]
Ping me when this is ready

**Pull request closed without merging**